### PR TITLE
Add BYOL feature to virtual machine.

### PR DESCRIPTION
**Pull request by hyonholee**

### CODE REVIEW

File: src/ResourceManagement/Compute/Compute.Tests/Helpers/Helpers.cs [hunk 1]
```diff
@@ -54,7 +54,7 @@ public static void DeleteIfExists(this IResourceGroupsOperations rgOps, string r
             }
         }
 
-        public static void ValidateVirtualMachineSizeListResponse(IPage<VirtualMachineSize> vmSizeListResponse)
+        public static void ValidateVirtualMachineSizeListResponse(IEnumerable<VirtualMachineSize> vmSizeListResponse)
```
Comment (huangpf) [inline src/ResourceManagement/Compute/Compute.Tests/Helpers/Helpers.cs]: This return type has changed quite some times. Is this time the final change? @stankovski

**Comment by huangpf** [comment]
LGTM.

File: src/ResourceManagement/Compute/Compute.Tests/Helpers/Helpers.cs (hunk 1 above)
Comment (stankovski) [inline src/ResourceManagement/Compute/Compute.Tests/Helpers/Helpers.cs]: So far we don't plan to change it again.

**Comment by stankovski** [comment]
@hyonholee if possible I'd like to wait with this PR until https://github.com/Azure/azure-sdk-for-net/pull/1748 is merged

**Comment by stankovski** [comment]
Retest this please

File: src/ResourceManagement/Compute/Microsoft.Azure.Management.Compute/ComputeManagementClient.json [hunk 2]
```diff
@@ -3521,6 +3521,10 @@
           "$ref": "#/definitions/VirtualMachineInstanceView",
           "readOnly": true,
           "description": "Gets the virtual machine instance view."
+        },
```
Comment (huangpf) [inline src/ResourceManagement/Compute/Microsoft.Azure.Management.Compute/ComputeManagementClient.json]: @hyonholee @stankovski 
Did you also update: https://github.com/Azure/azure-rest-api-specs/blob/master/arm-compute/2015-06-15/swagger/compute.json ?

**Pull request merged**